src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -32,6 +33,7 @@ export type RichTextEditorProps = {
     border?: boolean;
     updateValueOnChange?: boolean; // Only set to true when you are sure that performance isn't an issue
     toolbarWidth?: number;
+    hideExternalFloatingModals?: (editorId: string) => void;
```

Comment:
Add a test and story with this new prop

Reply:
I will, once this solution has been agreed upon.